### ISSUE TITLE
Cannot change cover art when already set

### ISSUE THREAD

**Issue by danielwolbach** [knuxify/eartag#47]
### Describe the issue

Changing the cover art of an mp3 file that already has one does not work.

[Example](https://user-images.githubusercontent.com/77749247/200179206-16c3044b-d356-415f-859c-4b5168b2b05f.webm)

**Comment by knuxify** [comment]
I guess the issue here is our hacky way of handling cover art in mp3 files (since some programs expect it to be in the front_cover variable, and some in the OTHER variable, I thought it would be OK to just keep things in the OTHER variable... it's probably better to set both, or only FRONT_COVER and remove OTHER if given the chance.)

I made 47143d4ced60aa6c72df24acb9c593c2a749879e to change this; could you test if this fixes the issue?

**Comment by knuxify** [comment]
(Related: #31)

**Comment by knuxify** [comment]
(Also unrelated, but it looks like the translation doesn't get fully loaded, which is something I've already been notified about via e-mail, but forgot to patch it... 000db357e4dfc0443addecf2c0a405f974ea3c00 might fix that, feel free to test that too.)

**Comment by octowaddle** [comment]
I tried building it in GNOME Builder, however I get this error:
```
Python built without locale support, some translations may not work!
Traceback (most recent call last):
  File "/app/bin/eartag", line 56, in <module>
    from eartag import main
  File "/app/share/eartag/eartag/main.py", line 38, in <module>
    from .common import is_valid_music_file
  File "/app/share/eartag/eartag/common.py", line 34, in <module>
    import magic
```

Unfortunately, I don't have any knowledge of Python.

(Also by the way; the path to the python modules in the Flatpak manifest is incorrect. It needs to be `"data/flatpak-python-modules.json"` instead of `"data/flatpak-python3-modules.json"`.

**Comment by danielwolbach** [comment]
I was able to compile and run, unfortunately the bug remains.
I have done stuff with music metadata once and I know that you can add multiple cover arts to a single file - maybe instead of swapping the cover art the app just adds a new a one? Most music players only show the primary (the "first") cover art. This is just a guess of course.

**Comment by knuxify** [comment]
Could you send me the files that you're testing with so that I can investigate this further? My email is knuxify (at) gmail (dot) com.

**Comment by knuxify** [comment]
I just release Ear Tag 0.3.0, which also contains a rewrite of the MP3 backend. Could you check if the issue still persists?

**Comment by danielwolbach** [comment]
I tried it and it works now, thank you very much! However, a new issue is now occurring: after I select the album art from the file picker, the change is not reflected in the UI. Only after I save, close and reopen the app the picture is changed.

**Comment by knuxify** [comment]
> However, a new issue is now occurring: after I select the album art from the file picker, the change is not reflected in the UI. Only after I save, close and reopen the app the picture is changed.

Yup, can confirm this happens. e59204f should fix it.
